Library/Formula/pmd.rb

```diff
@@ -2,23 +2,26 @@
 
 class Pmd < Formula
   homepage "http://pmd.sourceforge.net/"
-  url "https://downloads.sourceforge.net/project/pmd/pmd/5.1.3/pmd-src-5.1.3.zip"
-  sha1 "da6023d73e078444a7b76e8a7a0cfbeb421221e4"
+  url "https://downloads.sourceforge.net/project/pmd/pmd/5.3.2/pmd-src-5.3.2.zip"
+  sha1 "036cdb7583de3ea0d24e7486bf7b9f9f14bfb1ec"
+
+  depends_on "maven" => :build
+
+  patch :DATA
 
   def install
-    rm Dir["bin/*.{bat,cmd,dll,exe}"]
+    system "mvn", "clean", "package"
 
-    doc.install "LICENSE", "NOTICE", "ReadMe.txt", "licences", Dir["docs/*"]
-    libexec.install "bin", "etc", "lib"
+    doc.install "LICENSE", "NOTICE", "README.md"
 
-    bin.install_symlink "#{libexec}/bin/run.sh" => "pmd"
+    libexec.install "pmd-dist/target/pmd-bin-#{version}/pmd-bin-#{version}/bin", "pmd-dist/target/pmd-bin-#{version}/pmd-bin-#{version}/lib"
 
-    # the run script references paths which don't account for the
-    # file structure of this brew.
+    bin.install_symlink "#{libexec}/bin/run.sh" => "pmd"
     inreplace "#{libexec}/bin/run.sh", "${script_dir}/../lib", "#{libexec}/lib"
   end
 
   def caveats; <<-EOS.undent
+    Once pmd is installed, maven isn't needed anymore.
```

Comment:
@mikemcquaid the reason didn't seem reasonable since the it's not only build time. From @jacknagel answer I get that this is expected from brew. I'll remove it in light of that. Sorry if I'm getting on your nerves, and thanks for your time.